MediaBrowser.Model/LiveTv/TunerHostInfo.cs

```diff
@@ -31,5 +31,7 @@ public TunerHostInfo()
         public int TunerCount { get; set; }
 
         public string UserAgent { get; set; }
+
+        public bool IgnoreDts { get; set; } = true;
```

Comment:
`AllowHWTranscoding` is set in the constructor, so might as well keep it consistent

Reply:
Fixed